xwiki-rendering-api/src/test/java/org/xwiki/rendering/listener/chaining/AbstractChainingListenerTest.java

```diff
@@ -44,7 +44,7 @@
  */
 class AbstractChainingListenerTest
 {
-    private static class AbstractChild extends AbstractChainingPrintRenderer
+    private static final class AbstractChild extends AbstractChainingPrintRenderer
```

Comment:
This class cannot be final because `Child2` inherits from it. This change doesn't compile.

Reply:
@michitux that was my mistake